bubble_sort.rb

```diff
@@ -0,0 +1,37 @@
+def bubble_sort(arr)
+  l = arr.length
+  return arr if l <= 1
+
+  change = true
+
+  while change
+    i = 0
+    while i < l - 1
+      if arr[i] > arr[i + 1]
+        arr[i], arr[i + 1] = arr[i + 1], arr[i]
+      else
+        change = false
+      end
+      i += 1
+    end
+  end
+  arr
+end
```

Comment:
- [ ] Bubble_sort should also sort an array in increasing order
``` for example, if I pass the following array [8, 7, 6, 5, 4, 3, 2, 1, 9, 10]  it should sort in this manner [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]```

Reply:
Done!